devel/libptytty/Portfile

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+
+name                libptytty
+version             2.0
+categories          devel
+license             GPL-2+
+maintainers         {gmail.com:emanuele.giaquinta @exg} openmaintainer
+description         library for OS-independent pseudo-tty management
+long_description    ${description}
+platforms           darwin
```

Comment:
While this is minor, `platforms darwin` is now the default. So you can remove that declaration.

Reply:
Removed, thanks.